app/addons/documents/changes/components.react.jsx

```diff
@@ -199,165 +205,102 @@ var AddFilterForm = React.createClass({
       this.props.addFilter(this.state.filter);
       this.setState({ filter: '', error: false });
     }
-  },
+  }
 
-  componentDidMount: function () {
+  componentDidMount () {
     this.focusFilterField();
-  },
+  }
 
-  componentDidUpdate: function () {
+  componentDidUpdate () {
     this.focusFilterField();
-  },
+  }
 
-  focusFilterField: function () {
+  focusFilterField () {
     ReactDOM.findDOMNode(this.refs.addItem).focus();
-  },
-
-  onChangeFilter: function (e) {
-    this.setState({ filter: e.target.value });
-  },
+  }
 
-  inputClassNames: function () {
+  inputClassNames () {
     var className = 'js-changes-filter-field';
```

Comment:
The rationale was that the `js-` prefixes would indicate the class only has significance for the JS code - especially test code - and not be used for any styling. It seemed a good idea at the time :)- but it is being used by a bunch of tests.